# Name of the location
int32 databaseID

---
# coordinates
bool exists # true if the name of the location exists in the map
string objname
string category
geometry_msgs/Point32 base_coordinates  # in /map ref frame
geometry_msgs/Pose    arm_coordinates	# in /base_link ref frame

